migration_toolkit_for_containers/installing-mtc-restricted.adoc

```diff
@@ -37,6 +37,24 @@ include::modules/migration-configuring-proxies.adoc[leveloffset=+2]
 
 For more information, see xref:../networking/enable-cluster-wide-proxy.adoc#nw-proxy-configure-object_config-cluster-wide-proxy[Configuring the cluster-wide proxy].
 
+[id="migration-rsync-root-non-root_{context}"]
+== Running Rsync as either root or non-root
+
+[IMPORTANT]
+====
+This section applies only when you are working with the OpenShift API, not the web console.
+====
+
+OpenShift environments have the `PodSecurityAdmission` controller enabled by default. This controller requires cluster administrators to enforce Pod Security Standards by means of namespace labels. All workloads in the cluster are expected to run one of the following Pod Security Standard levels: `Privileged`, `Baseline` or `Restricted`. Every cluster has its own default policy set.
+
+To guarantee successful data transfer in all environments, {mtc-full} ({mtc-short}) 1.7.5 introduced changes in Rsync pods, including running Rsync pods as non-root user by default. This ensures that data transfer is possible even for workloads that do not necessarily require higher privileges. This change was made because it is best to run workloads with lowest level of privileges possible.
```

Comment:
```suggestion
To guarantee successful data transfer in all environments, {mtc-full} ({mtc-short}) 1.7.5 introduced changes in Rsync pods, including running Rsync pods as non-root user by default. This ensures that data transfer is possible even for workloads that do not necessarily have access to higher privileges. This change was made because it is best to run workloads with lowest level of privileges possible.
```

?

Reply:
also it's not clear....does this mean rsync can run using `restricted` permissions or not?  running non-root is one aspect, but it doesn't tell me whether any other privileges are still needed that prevent running rsync pods in a restricted NS.

Reply:
@stillalearner -- are other privileges needed?